101-azure-api-for-fhir/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+     "serviceName":{
+        "value": "GEN-UNIQUE"
+     },
+     "location":{
+        "value": "GEN-UNIQUE"
```

Comment:
Instead of GEN-UNIQUE for location, it looks like you'll need to set it to one of supported locations for the service. You could use "westus".

Reply:
actually westus2

Reply:
if the service is globally available, just remove this and use a defaultValue in template